### PR TITLE
1.1.2 - Updated React dependencies (React 16 compatibility)

### PR DESCRIPTION
**Pull request by JaredBoone**

### CODE REVIEW

**Comment by JaredBoone** [comment]
Travis CI tests failed

**Comment by JaredBoone** [comment]
npm ERR! peerinvalid Peer enzyme@2.9.1 wants react@0.13.x || 0.14.x || ^15.0.0-0 || 15.x
npm ERR! peerinvalid Peer react-test-renderer@15.6.2 wants react@^15.6.2

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,9 @@
 language: node_js
 node_js:
   - "6"
-  - "4"
-  - "node"
+  - "8"
```
Comment (isuttell) [inline .travis.yml]: Are we not going to use `lts/*` here?

Comment (JaredBoone) [reply .travis.yml]: Trevor parses the .travis.yml file differently than Travis CI does. It only accepts "stable", "node", or "X.Y.Z" (not "lts", etc). "stable" and "node" are equivalent, and they are determined by downloading and parsing nodejs.org/dist, to find the latest version with an even minor number. Also, Trevor ignores our minor and patch version specifications, so "X.Y.Z" does the equivalent of "nvm use X".

**Pull request merged**